src/main/java/mx/paradigmmasters/bastagui/view/JoinView.java

```diff
@@ -17,8 +17,12 @@ public JoinView(MainControl control) throws HeadlessException {
         this.setVisible(true);
         this.setDefaultCloseOperation(EXIT_ON_CLOSE);
         this.setLayout(new GridLayout(4,1));
+        this.setSize(400, 400);
```

Comment:
No es necesario poner un tamaño de ventana, al final del constructor se usa `this.pack()` para que Swing calcule automáticamente el tamaño de ventana ideal, por lo que esta línea es inutil.